src/components/WebexAvatar/WebexAvatar.scss

```diff
@@ -7,8 +7,26 @@
 
   position: relative;
 
+  border-radius: 50%;
+
+  background-color: transparent;
+
+  .initials {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    height: 100%;
+
+    font-size: 0.875rem;
+    color: var(--wxc-avatar--text-color);
+  }
+
   img {
     border-radius: 50%;
+    width: 100%;
```

Comment:
does the parent still need `display: flex`? it seems that both its children use `height: 100%`